controllers/components/navbar.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+
+export default class Admin_Navbar extends React.Component {
+    constructor( props ){
+        super( props );
+        this.state = {
+            team: '',
+            teamLink: ''
+        }
+        this._handleSearch = this._handleSearch.bind( this );
+    }
+
+    componentDidMount() {
+        const that = this;
+        const user = JSON.parse( atob( localStorage.token.split( '.' )[ 1 ] ) );
+
+        // TODO make this fetch function update all navbar details
+        fetch( '/users/get-one-user/' + user.id )
+        .then( function( res ) {
+            if ( !res.ok ) return alert( 'failed fetching user' );
+            res.json().then( function( user ) {
+                that.setState({
+                    team: user.team
+                });
+                if ( that.state.team ) { fetchTeamName( user.team ); }
+            });
+        }).catch(function( err ) { alert( err ) } );
+
+        // Fetch team name of user
+        function fetchTeamName( id ) {
+            fetch( '/teams/get-one-team/' + id )
+            .then( function( res ) {
+                if ( !res.ok ) return alert( 'failed fetching team' );
+                res.json().then( function( team ) {
+                    that.setState({
+                        teamLink: team.teamName
+                    });
+                });
+            }).catch( function( err ) { alert( err ) } );
+        }
+    }
+
+    _logoutUser( e ) {
+        e.preventDefault();
+        localStorage.removeItem( 'token' );
+        window.location = '/';
+    }
+
+    _handleSearch( e ) {
+        const form = document.forms[ 's' ];
+        const search_for = prompt( 'Search for?\ntype: \'user\' or \'team\'' );

```

Comment:
Use double quotes here so you can use single quotes without escaping.

Reply:
Yeh, good call.